packages/flutter/lib/physics.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.8
+// @dart = 2.9
```

Comment:
In the other NNBD PRs this annotation was completely removed once a file was migrated. Why keep it here?

Reply:
Oops, sorry. It was old code I didn't change.